src/router-middleware.js

```diff
@@ -26,8 +26,8 @@ export default function routerMiddleware($state) {
       return next({
         type: STATE_CHANGE_SUCCESS,
         payload: {
-          currentState: $state.current,
-          currentParams: $state.params,
+          currentState: $state.$current.name,
+          currentParams: $state.$current.params,
```

Comment:
Oh, sorry. I forgot to rollback this line while experimenting with State ($current), instead of StateDeclaration (current).